lib/monthly.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module Declarations
+  module Prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
```

Comment:
The service name `MonthlyVatReturnService` does not use an action verb. Consider renaming it to reflect the action it performs, such as `CalculateMonthlyVatReturnService` or `GenerateMonthlyVatReturnService`.